fuel-core/src/model.rs

```diff
@@ -1,4 +1,13 @@
-pub mod coin;
-pub mod fuel_block;
+use async_graphql::Enum;
+pub use fuel_core_interfaces::models::{
+    BlockHeight, Coin, CoinStatus, FuelBlock, FuelBlockDb, FuelBlockHeader,
+};
 
 pub type Hash = [u8; 32];
+
+#[derive(Enum, Copy, Clone, Eq, PartialEq)]
+#[graphql(remote = "CoinStatus")]
+pub enum SchemaCoinStatus {
```

Comment:
Prettify the naming of the schema by aliasing imports

```suggestion
    BlockHeight, Coin, CoinStatus as CoinStatusModel, FuelBlock, FuelBlockDb, FuelBlockHeader,
};

pub type Hash = [u8; 32];

#[derive(Enum, Copy, Clone, Eq, PartialEq)]
#[graphql(remote = "CoinStatusModel")]
pub enum CoinStatus {
```

Reply:
Yep